recipes/cprocessing/all/conanfile.py

```diff
@@ -18,10 +18,8 @@ class CProcessingConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/maksmakuta/CProcessing"
     topics = ("processing", "opengl", "sketch", "header-only")
-
     package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
-    no_copy_source = True
```

Comment:
Isn't it the other way around? As we don't have patches, we can set `no_copy_source` to True to avoid unnecessary copies, no?

Reply:
there is a patch, a replace_in_file()

Reply:
Oh I missread your PR comment and thought that there werent, thanks for the clarification :)